src/modifiers.rs

```diff
@@ -1,30 +1,49 @@
-//! Defines a series of convenience modifiers for editing Responses
+//! This module defines a series of convenience modifiers for changing
+//! Responses.
 //!
-//! Modifiers can be used to edit Responses through the owning
-//! method `set` or the mutating `set_mut`, both of which are
-//! defined in the `Set` trait.
+//! Modifiers can be used to edit `Response`s through the owning method `set`
+//! or the mutating `set_mut`, both of which are defined in the `Set` trait.
 //!
-//! Instead of having a combinatorial explosion of Response methods
-//! and constructors, this provides a series of modifiers that
-//! can be used through the `Set` trait.
+//! Instead of having a combinatorial explosion of Response methods and
+//! constructors, this provides a series of modifiers that can be used through
+//! the `Set` trait. This is more extensible as it allows you to combine
+//! arbitrary modifiers without having a massive number of Response
+//! constructors.
```

Comment:
I think the whole paragraph could be replaced with "For Iron, the `Modifier` interface offers extensible and ergonomic `Response` creation while avoiding the introduction of many highly specific `Response` constructors."